package-lock.json

```diff
@@ -22,9 +22,9 @@
       },
       "dependencies": {
         "commander": {
-          "version": "4.0.1",
```

Comment:
Is it expected `npm install` to generate so many diffs, or did I b0rk something?

Reply:
Don't worry about it. Please, develop against the `dev` branch in order to be sure to have the latest version of the dependencies and of the code.